features/preferred_profile.feature

```diff
@@ -95,7 +103,7 @@ Feature: Preferred Profiles
         progress
 
       imports:
-        - pretty.yml
+        - pretty.php
```

Comment:
This is nice, I did not realize that you could import php files from yaml files 😄

Reply:
AHAH, and you can import yaml files from php config :)